modelforge/dataset/dataset.py

```diff
@@ -206,16 +206,17 @@ def __getitem__(self, idx: int) -> Dict[str, torch.Tensor]:
         positions = self.properties_of_interest["positions"][
             series_atom_start_idx:series_atom_end_idx
         ]
-        E = self.properties_of_interest["E"][
-            idx
-        ]  # NOTE: upgrading to float64 to avoid precision issues
+        E = self.properties_of_interest["E"][idx]
+        F = self.properties_of_interest["F"][series_atom_start_idx:series_atom_end_idx]
```

Comment:
This is assuming that all datasets have forces.  This will fail for qm9.    I had made F and Q optional in the PropertyNames class...we could always change the data loaders to just pack the force with zeroes if not available, or we I guess need to make this some how optional here (maybe this will look over the PropertyNames to know what is available first?)

Reply:
Maybe this ends up just going back to @jchodera point of dumping QM9 altogether!  But, I think we still want some ability to be able to comparing training with and without forces for comparison.  So I'm not sure how best to handle this. Maybe we can discuss Tuesday morning.

Reply:
Actually, the tests are passing just fine with QM9. In that case all forces are zero, but I think that is fine.

Reply:
There will be an error thrown when trying to calculate the loss, since there are no QM forces, though:
https://github.com/choderalab/modelforge/blob/f98cd99f192ecbea32c428cc975e6f506478ff2e/modelforge/train/training.py#L49

One possible strategy is also to initialize these with zeros. Then no error is thrown but the force contributions are ignored in the loss calculation.